### PR TITLE
SDCORE-781: AMF should send PDU rejection message when SMF doesn't respond

### PR DESCRIPTION
**Pull request by vthiruveedula**

### CODE REVIEW

File: gmm/handler.go [hunk 1]
```diff
@@ -583,6 +585,9 @@ func HandleInitialRegistration(ue *context.AmfUe, anType models.AccessType) erro
 
 	if len(ue.AllowedNssai[anType]) == 0 {
 		gmm_message.SendRegistrationReject(ue.RanUe[anType], nasMessage.Cause5GMM5GSServicesNotAllowed, "")
+		ngap_message.SendUEContextReleaseCommand(ue.RanUe[anType], context.UeContextN2NormalRelease,
+			ngapType.CausePresentNas, ngapType.CauseNasPresentNormalRelease)
+		ue.Remove()
```
Comment (thakurajayL) [inline gmm/handler.go:590]: Can you create call flow for this problem and fix ?

Comment (gab-arrobo) [reply gmm/handler.go]: @thakurajayL, it looks like this comment is outdated. Correct?

Comment (thakurajayL) [reply gmm/handler.go]: I would prefer to take code as is. Valid fix to cleanup N1 tunnel.

Comment (gab-arrobo) [reply gmm/handler.go]: @thakurajayL, so, should we approve this PR?

**Comment by gab-arrobo** [comment]
Hi @vthiruveedula, is this PRs still valid? or should I close it?

**Pull request closed without merging**